doc/nrf/app_dev/device_guides/nrf54l/building_nrf54l.rst

```diff
@@ -0,0 +1,138 @@
+
+Complete the following steps:
+
+.. tabs::
+
+   .. group-tab:: Using VPR Launcher
+
+      VPR launcher allows to start the FLPR core.
+      When building for the ``nrf54l15dk/nrf54l15/cpuflpr`` target, the launcher loads the minimal sample to the application core by default.
+      The primary function of this image is to transfer the FLPR code to the designated region (if necessary) and initiate the FLPR core.
+
+      To build and flash both images, run the following command:
+
+      .. code-block:: console
+
+         west build -p -b nrf54l15dk/nrf54l15/cpuflpr
+         west flash
+
+   .. group-tab:: Using application that supports multi-image builds
+
+      If your application involves creating custom images for both the application core and the FLPR core, make sure to disable the VPR Launcher.
+      You can do this by disabling the ``SB_CONFIG_VPR_LAUNCHER`` option when building for the FLPR target.
+      For more details, see :ref:`how to configure Kconfig <configuring_kconfig>`.
+
+      To build and flash both images, run the following command:
+
+      .. code-block:: console
+
+         west build -p -b nrf54l15dk/nrf54l15/cpuflpr -- -DSB_CONFIG_VPR_LAUNCHER=n
+         west flash
+
+Separate images
+---------------
+
+You can build and program application sample and the FLPR sample as separate images using the |nRFVSC| or command line.
+To use nRF Util, see `Programming application firmware on the nRF54L15 SoC`_.
+Depending on the selected method, complete the following steps:
+
+.. tabs::
+
+   .. group-tab:: nRF Connect for VS Code
+
+      .. note::
+
+         The Visual Studio Code extension currently offers experimental support for the nRF54L15's FLPR core.
+         Certain features, particularly debugging, may not function as expected.
+
+      .. include:: /includes/vsc_build_and_run.txt
+
+      3. Build the application image by setting the following options:
+
+         * Board target to ``nrf54l15dk/nrf54l15/cpuapp``.
+         * Choose either ``nordic-flpr`` or ``nordic-flpr-xip`` snippet depending on the FLPR image target.
+         * System build to :guilabel:`No sysbuild`.
+
+         For more information, see :ref:`cmake_options`.
+
+      #. Build the FLPR image by setting the following options:
+
+         * Board target to ``nrf54l15dk/nrf54l15/cpuflpr`` (recommended) or ``nrf54l15dk/nrf54l15/cpuflpr/xip``.
+         * System build to :guilabel:`No sysbuild`.
+
+         For more information, see :ref:`cmake_options`.
+
+   .. group-tab:: Command line
+
+      1. |open_terminal_window_with_environment|
+      #. Build the application core image, and based on your build target include the appropriate snippet:
+
+         .. code-block:: console
+
+            west build -p -b nrf54l15dk/nrf54l15/cpuapp -S nordic-flpr --no-sysbuild
+
+      #. Flash the application core image by running the following command:
+
+         .. code-block:: console
+
+            west flash
```

Comment:
```suggestion
      #. Program the application core image by running the `west flash` command :ref:`without --erase <programming_params_no_erase>`.
```
"Flash" is not allowed as per https://nordicsemi.atlassian.net/wiki/spaces/TECHDOC/pages/120298016/Terms+abbreviations+and+spelling+rules#Terms,abbreviations,andspellingrules-P